packages/panzer/adapters-stk/src/stk_interface/Panzer_STK_ExodusReaderFactory.cpp

```diff
@@ -249,6 +271,11 @@ void STK_ExodusReaderFactory::setParameterList(const Teuchos::RCP<Teuchos::Param
 
    restartIndex_ = paramList->get<int>("Restart Index");
 
+   {
+     const auto fileType = paramList->get<std::string>("File Type");
+     isExodus_ = fileType == "Exodus" ? true : false;
```

Comment:
Is the `? true : false` needed?

Reply:
yikes! how did that sneak in? I'll fix it.